draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1296,9 +1296,10 @@ Truncating:
 
 Error Handling:
 : If a system overruns the generator by requesting too many UUIDs
-  within a single system time interval, the UUID service SHOULD either
+  within a single system time interval, the UUID service can either
   return an error, or stall the UUID generator until the system clock
-  catches up, and MUST NOT return knowingly duplicate values.
+  catches up, and MUST NOT return knowingly duplicate values due to
+  counter rollover.
```

Comment:
@LiosK I saw your comment on #114, I think this works to keep it MUST NOT but specifically say "due to counter rollover", so it excludes other edge cases such as the clock rolling back. I think that aligns with the intention of the text anyway, which is all about counters.